gammapy/spectrum/utils.py

```diff
@@ -13,19 +16,254 @@
            'np_to_pha',
            ]
 
-# TODO: remove functions now that we have classes!
 
+def find_log_centers(bin_edges):
+    """Compute energy bin log centers.
+
+    Bin center = sqrt(low_edge * high_edge)
+    """
+    return np.sqrt(bin_edges[:-1] * bin_edges[1:])
+
+
+def find_log_edges(bin_centers):
+    """Compute equally log-spaced energy bin edges
```

Comment:
For this, it would be good to have one or two examples in an examples section:
```
Examples
--------
```
This should be in the user-visible docs. Is this a public or private function?